packages/xo-web/src/xo-app/hub/recipes/recipe-form.js

```diff
@@ -100,6 +108,78 @@ export default decorate([
           value={value.nbNodes}
         />
       </FormGrid.Row>
+      <FormGrid.Row>
+        <label>
```

Comment:
use `htmlFor` for label

Reply:
This works too, I'm not sure there's a benefit of using `htmlFor`instead?